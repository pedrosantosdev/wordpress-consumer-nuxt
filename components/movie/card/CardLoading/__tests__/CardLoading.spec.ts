import { render } from '@testing-library/vue'
import CardLoadingVue from '@/components/movie/card/CardLoading/CardLoading.vue'

describe('CardLoadingVue', () => {
	it('should render properly', () => {
		const wrapper = render(CardLoadingVue)
		expect(wrapper.html()).toMatchSnapshot()
	})
})
