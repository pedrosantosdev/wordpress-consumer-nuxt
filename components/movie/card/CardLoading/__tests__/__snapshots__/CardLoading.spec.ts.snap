// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CardLoadingVue > should render properly 1`] = `
"<div class=\\"lg:w-96 w-full sm:h-64 md:h-auto flex flex-col sm:flex-row p-2 sm:p-4 rounded-2xl shadow-lg gap-5 select-none dark:bg-gray-800 bg-white dark:bg-gray-800 dark:text-white\\">
  <div class=\\"sm:w-72 h-52 sm:h-full rounded-xl animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
  <div class=\\"flex flex-1 flex-col gap-5 sm:p-2 sm:pt-0\\">
    <div class=\\"flex flex-1 flex-col gap-3\\">
      <div class=\\"w-full h-16 rounded-2xl animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
      <div class=\\"w-full h-16 rounded-2xl animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
    </div>
    <div class=\\"flex mt-auto gap-3\\">
      <div class=\\"w-20 h-8 rounded-full animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
      <div class=\\"w-20 h-8 rounded-full animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
      <div class=\\"w-20 h-8 rounded-full ml-auto animate-pulse bg-gray-200 dark:bg-gray-600\\"></div>
    </div>
  </div>
</div>"
`;
